cdap-data-fabric-tests/src/test/java/co/cask/cdap/data/stream/MultiLiveStreamFileReaderTestBase.java

```diff
@@ -46,11 +48,12 @@
   @Test
   public void testMultiFileReader() throws Exception {
     String streamName = "multiReader";
+    Id.Stream streamId = Id.Stream.from(Constants.DEFAULT_NAMESPACE, streamName);
```

Comment:
Possible to not use ``DEFAULT_NAMESPACE`` here? Or is that not possible since namespacing is not done yet?